### PR TITLE
add dolphin

### PR DESCRIPTION
**Pull request by vladikoff**
@miketaylr OR ANYONE ELSE please r+

### CODE REVIEW

**Comment by wesbos** [comment]
wat

**Comment by vladikoff** [comment]
> wat

SOLD.

**Comment by ralphholzmann** [comment]
forgot this existed

**Comment by vladikoff** [comment]
r=@WESBOS,@RALPH

landed. enjoy.

**Comment by wesbos** [comment]
now we gotta wait for @gf3 to wake up so he can reboot botty

**Comment by vladikoff** [comment]
all bots are dead, but the json is available for consumption.

File: wat.json [hunk 1]
```diff
@@ -241,5 +241,6 @@
    "I'm gonna get ride of this non-sense-shiny-grunt-do-as-i-wish-gonna-be-obsolete-in-a-few-years-losing-time with a bunch of python scripts",
    "Hire me for Google. I know web building.pls",
    "Babel is probably the new Dojo.",
-   "Wish we had riotjs native cuz reactjs sucks like Java swing"
+   "Wish we had riotjs native cuz reactjs sucks like Java swing",
+   "instead of clicking on the tabs like a dolphin, you just over on the 'toppest' of each tab and dive in"
```
Comment (jkanywhere) [inline wat.json]: `over` or `hover`?

Comment (vladikoff) [reply wat.json]: I'm just a messenger of this WAT

**Comment by miketaylr** [comment]
needs tests

**Comment by Brodingo** [comment]
🐬 thank 🐬 god 🐬 this 🐬 thing 🐬 still 🐬 exists 🐬

File: wat.json (hunk 1 above)
Comment (mathiasbynens) [reply wat.json]: over adds to the watness

Comment (BBonifield) [reply wat.json]: 🐬 🦅 🦁 , 🐻 .

**Comment by pifantastic** [comment]
23pe3f

File: wat.json (hunk 1 above)
Comment (jkanywhere) [reply wat.json]: `watness` or `wetness` ?

**Comment by kieran** [comment]
image

**Comment by gf3** [comment]
i love open sorces

**Pull request merged**